### DISCUSSION TITLE
How do i use shaders

### DISCUSSION THREAD

**Discussion by Theoyeah** [DEMOLITIONDON96/Demolition-Engine#109] [Lua Q&A]
Idk how to do it in my engine

**Comment by RealDealJinx** [comment]
i can add them if you want

**Reply by Theoyeah** [reply]
i already added the shaders, but how do i apply them in a song?

**Reply by RealDealJinx** [reply]
```lua
function onCreatePost()
    shader(camera)
end
```

example
```lua
function onCreatePost()
    vcrEffect('hud',0.4,true,true,true)
end
```

**Comment by ghost** [comment]
https://github.com/DEMOLITIONDON96/Demolition-Engine/wiki/New-Lua-Content#shaders
or
https://github.com/DEMOLITIONDON96/Demolition-Engine/wiki/Shaders.#yeah-shaders
